google/cloud/storage/examples/storage_object_samples.cc

```diff
@@ -259,7 +259,18 @@ void GetObjectMetadata(google::cloud::storage::Client client,
 
     std::cout << "The metadata for object " << object_metadata->name()
               << " in bucket " << object_metadata->bucket() << " is "
-              << *object_metadata << "\n";
+              << "\n CacheControl : " << object_metadata->cache_control()
```

Comment:
The message is now technically incorrect, this is a selection of the metadata, not the full metadata, it is missing (for example) the storage class and the custom `metadata()`.   Maybe say "Selected metadata attributes for object "?